src/app/projects/projlist-filters/projlist-filters.component.html

```diff
@@ -2,23 +2,40 @@
   <div class="search-container" id="applist-filters">
     <fieldset class="additional-filters" (click)="$event.stopPropagation()">
       <div class="form-group typeahead-container">
-        <label for="applicantInput">Search Active Land and Water Planning Projects by Title</label>
-        <p class="additional-text">Additional land and water planning projects will be added to the map as they are initiated.</p>
-          <input type="text" class="form-control gtm-filter-applicant "
-          id="applicantInput" [(ngModel)]="_applicantFilter" [ngbTypeahead]="applicantSearch">
+        
+        <div class="project-search-title-row">
+          <label for="applicantInput">Search Projects by Title</label>
+          <button class="project-type-filter-button" id="show-hide-search" (click)="handleShowHideFilter('search')">
+            <i [ngClass]="{'collapsed': !searchOpen}" class="material-icons filter-collapse-button">
+              {{filterIconText}}
+            </i>
+          </button>
         </div>
-        <button type="button" class="btn btn-primary" title="Apply Filters" id="applyButton" (click)="applyAllFilters();"
-        [disabled]="loading"><i class="material-icons" aria-hidden="true">search</i>Search</button>
-      </fieldset>
 
-    </div>
-    <div class="instructions map-instructions">You can also <a class="paragraph-link" routerLink="/projects-list">go to list view of projects.</a></div>
+        <div *ngIf="searchOpen">
+          <p class="additional-text">Additional land and water planning projects will be added to the map as they are initiated. You can also 
```

Comment:
In the interest of accessibility, could we please make the text about the "list view page" more perceivable to screenreaders? Maybe an sr-only label that says something like: "People using assistive devices are advised to search on the list view page," and then provide the link.

Reply:
Sure, I can add a message about that.